### PR TITLE
flif 0.4

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

~`resource` blocks may require updates.~

### CODE REVIEW

**Review by iMichka — CHANGES_REQUESTED** [review]
Can you check the license too?

File: Formula/flif.rb [hunk 1]
```diff
@@ -2,10 +2,10 @@ class Flif < Formula
   desc "Free Loseless Image Format"
   homepage "https://flif.info/"
   # When updating, please check if FLIF switched to CMake yet
```
Comment (carlocab) [inline Formula/flif.rb:4]: I think they use CMake now. You can try
```ruby
system "cmake", "-S", "src", "-B", "build"
system "cmake", "--build", "build"
system "cmake", "--install", "build"
```

**Comment by chenrui333** [comment]
<details>
<summary>license ref</summary>

```
/*
 FLIF - Free Lossless Image Format
 Copyright (C) 2010-2016  Jon Sneyers & Pieter Wuille, LGPL v3+

 This program is free software: you can redistribute it and/or modify
 it under the terms of the GNU Lesser General Public License as published by
 the Free Software Foundation, either version 3 of the License, or
 (at your option) any later version.

 This program is distributed in the hope that it will be useful,
 but WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU General Public License for more details.

 You should have received a copy of the GNU Lesser General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
*/
```


</details>

File: Formula/flif.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/flif.rb:4]: Tried in my local, does not build successfully.

>   file INSTALL cannot copy file
  "/tmp/flif-20211206-95109-192d1ig/FLIF-0.4/build/libflif.0.dylib" to
  "/usr/local/lib/libflif.0.dylib": Operation not permitted.

Comment (carlocab) [reply Formula/flif.rb:4]: oops, that was supposed to be
```ruby
system "cmake", "-S", "src", "-B", "build", *std_cmake_args
system "cmake", "--build", "build"
system "cmake", "--install", "build"
```

**Comment by chenrui333** [comment]
Good for the merge?

**Review by iMichka — APPROVED** [review]

**Pull request closed without merging**